Woof/Woof/View/OwnerProfileView/EditOwnerInformationView.swift

```diff
@@ -0,0 +1,92 @@
+import SwiftUI
+
+/// A view to edit the owner information.
+struct EditOwnerInformationView: View {
+    /// The name of the owner
+    @Binding var name: String
+
+    /// The surname of the owner
+    @Binding var surname: String
+
+    /// The phone of the owner
+    @Binding var phone: String
+
+    /// The address of the owner
+    @Binding var address: String
+
+    var body: some View {
+        VStack {
+            Text("Edit your information")
+            TextField("Name", text: $name)
```

Comment:
nit:
Is it possible to improve the user experience?
```suggestion
            TextField("Enter your name", text: $name)
```